### ISSUE TITLE
Solo `make` command doesn't usually work (generate help text) on Windows

### ISSUE THREAD

**Issue by pjbull** [drivendata/cookiecutter-data-science#221]
Encountered this in a downstream project:
https://github.com/drivendataorg/sfp-cervical-biopsy-runtime/issues/18

Fix was to just remove the `more` command since it's not available on most windows bash shells:
https://github.com/drivendataorg/sfp-cervical-biopsy-runtime/pull/19

Expected behavior: running `make` generates help text

- [x] ensure help text works as expected
- [ ] #333
- [x] consider adding test case
  - [x] for make generating help text (or at least, not failing on the github actions windows runner)

**Comment by chrisjkuch** [comment]
So this is an error, but it turns out the Makefile help command doesn't work at all on any platform... need to re-think this.

**Comment by chrisjkuch** [comment]
On current v1, this shows up as
```
process_begin: CreateProcess(NULL, dirname C:/Users/chris/dd/project_name/Makefile, ...) failed.
Makefile:7: pipe: No error
process_begin: CreateProcess(NULL, which conda, ...) failed.
Makefile:13: pipe: No error
process_begin: CreateProcess(NULL, uname, ...) failed.
Makefile:110: pipe: No such file or directory
'test' is not recognized as an internal or external command,
operable program or batch file.
"$(tput bold)Available rules:$(tput sgr0)"
ECHO is off.
'sed' is not recognized as an internal or external command,
operable program or batch file.
make: *** [Makefile:110: help] Error 255
```
if `sed` is not installed

**Comment by chrisjkuch** [comment]
Closed by #335
